modules/dnn/src/onnx/onnx_graph_simplifier.cpp

```diff
@@ -192,20 +186,18 @@ class GeluSubGraph : public Subgraph
     {
         if (Subgraph::match(net, nodeId, matchedNodesIds, targetNodesIds))
         {
-            bool withInitializer = isWithInitializer(matchedNodesIds);
-
             // Check Div[B=sqrt(2)]
-            float divisor = extractConstant(net, matchedNodesIds[0], 1, withInitializer);
+            float divisor = extractConstant(net, matchedNodesIds[0], 1);
             if (divisor - M_SQRT2 >= 1e-6)
```

Comment:
why don't we use fabs() here? it's not a part of your patch, but this place looks strange

Reply:
Sure. It is fixed with the use of fabs().